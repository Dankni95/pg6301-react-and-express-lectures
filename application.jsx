import React from "react";

export function Application({who}) {
  return <h1>Hello {who}</h1>;
}
