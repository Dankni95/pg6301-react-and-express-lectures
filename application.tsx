import * as React from "react";

export function Application({ who }: { who: string }) {
  return <h1>Hello {who}</h1>;
}
