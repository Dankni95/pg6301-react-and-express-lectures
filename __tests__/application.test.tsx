import React from "react";
import ReactDOM from "react-dom";
import { Application } from "../application";

describe("application", () => {
  it("renders as expected", () => {
    const element = document.createElement("div");
    ReactDOM.render(<Application who="React" />, element);
    expect(element).toMatchSnapshot();
  });
});
