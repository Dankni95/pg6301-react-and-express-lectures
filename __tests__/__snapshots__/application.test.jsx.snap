// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application renders as expected 1`] = `
<div>
  <h1>
    Hello React
  </h1>
</div>
`;
